app/assets/stylesheets/global.css.scss

```diff
@@ -53,3 +53,50 @@ button.button a {
   color: white;
   border-radius: 18px;
 }
+
+.header {
+  // text-align: center;
+  background: asset-url("red-panda.jpg") no-repeat center center;
+  background-size: cover;
+  padding: 4rem;
+
+  .main-header {
+    color: #fff;
+    font-size: 3.8rem;
+    text-align: right;
+  }
+}
+
+.header-subnav {
+  float: none;
+  position: relative;
+  top: 4rem;
+  text-align: center;
+  margin-bottom: 0;
+
+  li {
+    float: none;
+    display: inline-block;
+
+    a {
+      padding: 0.9rem 1rem 0.75rem;
```

Comment:
Properties should be ordered color, display, font-size, font-weight, letter-spacing, padding, text-transform